lib/cylc/scheduler.py

```diff
@@ -445,14 +467,15 @@ def configure(self):
                     self.suite, user_at_host)
             except RemoteJobHostInitError as exc:
                 self.log.error(str(exc))
+        self.old_user_at_host_set.clear()
 
         self.already_timed_out = False
         if self._get_events_conf(self.EVENT_TIMEOUT):
             self.set_suite_timer()
 
-        self.nudge_timer_start = None
-        self.nudge_timer_on = False
-        self.auto_nudge_interval = 5  # seconds
+        # self.nudge_timer_start = None
```

Comment:
These variables are only used in some commented out logic later on in the code, so I thought I would comment them out. I don't want to remove them as part of this change, however.